cfme/configure/tasks.py

```diff
@@ -162,10 +170,126 @@ def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
                             clear_tasks_after_success=clear_tasks_after_success)
 
 
-def wait_analysis_finished(task_name, task_type, delay=5, timeout='5M'):
+def are_all_tasks_finished(name, expected_num_of_tasks, task_type='container'):
+    """ Check if all tasks states are finished - if not, reload page"""
+
+    tabs_data = {
+        'container': {
```

Comment:
Is it container specific function?

Reply:
wait_analysis_finished & are_all_tasks_finished are related to Tasks.
Its written for containers SSA check , and on usage check for now only container Image are using it.

Reply:
**REQUIRED**
It should be renamed or implemented as more versatile, it shouldn't be only for containers. The second option is more preferable.

Reply:
OK I get it,
I add more task types to tabs_data on are_all_tasks_finished,  so it will be more versatile , and not only for containers.